### PR TITLE
Release 5.5.0

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

### Added

- Locations ajax for results output listing.
- Option to customize Date and Time fields Preview and Output format.
- Result Outputs can now be exported and imported using settings.
- Settings entries list now has better UX in settings.
- `getFormUsageLocation` helper now supports custom post type.
- HubSpot integration now works using `v3` API for all data except form submission as it is not supported.

### Fixed

- Input Range status variable will output max 2 decimals.
- Input Range settings pickers broken up/down buttons.

### Removed

- Any Server Side Render for the forms and results output.
- Result Output global settings option to setup custom frontend URL.

### CODE REVIEW

**Review by iobrado — APPROVED** [review]

**Review by iobrado — DISMISSED** [review]

File: src/Blocks/components/date/manifest.json [hunk 1]
```diff
@@ -91,5 +89,15 @@
 				"value": "datetime-local"
 			}
 		]
+	},
+	"formats": {
+		"date": {
+			"preview": "F j, Y",
+			"output": "Y-m-d"
+		},
+		"datetime-local": {
```
Comment (piqusy) [inline src/Blocks/components/date/manifest.json:98]: How come this one has `-local` suffix and the other one doesn't?

File: src/Blocks/components/form/assets-admin/transfer.js [hunk 2]
```diff
@@ -98,16 +98,17 @@ export class Transfer {
...
-		let output = items ? items.split(",") : [];
-
 		const {
 			value,
 			checked,
+			name,
 		} = event.target;
 
+		const button = document.querySelector(`${this.state.getStateSelector('field', true)}[${this.state.getStateAttribute('migrationType')}='${name}']`);
+		const items = button.getAttribute(this.state.getStateAttribute('migrationExportItems'));
```
Comment (piqusy) [inline src/Blocks/components/form/assets-admin/transfer.js:108]:
```suggestion
		const items = button?.getAttribute(this.state.getStateAttribute('migrationExportItems'));
```

File: src/Blocks/components/input/manifest.json [hunk 3]
```diff
@@ -158,23 +158,28 @@
...
 			"step": 1
 		},
 		"inputMin": {
 			"min":  0.01,
+			"max": 999999,
 			"step": 0.01
 		},
 		"inputMax": {
-			"min":  0.01,
+			"min": 0.01,
```
Comment (piqusy) [inline src/Blocks/components/input/manifest.json:176]:
```suggestion
			"min": 0,
```
Why couldn't this be 0?

File: src/Rest/Routes/Settings/TransferRoute.php [hunk 4]
```diff
@@ -115,12 +117,31 @@ public function routeCallback(WP_REST_Request $request)
...
+				$output[Transfer::TYPE_FORMS] = $this->transfer->getExportCpts($items);
+				$internalType = 'export';
+				break;
+			case SettingsTransfer::TYPE_EXPORT_RESULT_OUTPUTS:
+				$items = $params['items'] ?? [];
+
+				if (!$items) {
+					return \rest_ensure_response(
+						UtilsApiHelper::getApiErrorPublicOutput(
+							\esc_html__('Please click on the result outputs you want to export.', 'eightshift-forms'),
```
Comment (piqusy) [inline src/Rest/Routes/Settings/TransferRoute.php:129]:
```suggestion
							\esc_html__('Please click on the Result outputs you want to export.', 'eightshift-forms'),
```

**Review by piqusy — CHANGES_REQUESTED** [review]
Nice work 👏🏼 

Left a few suggestions

File: src/Integrations/Corvus/SettingsCorvus.php [hunk 5]
```diff
@@ -231,252 +237,275 @@ static function ($option) use ($selectedStoreId) {
...
+								],
+								[
+									'component' => 'checkboxes',
+									'checkboxesFieldLabel' => '',
+									'checkboxesName' => UtilsSettingsHelper::getSettingName(self::SETTINGS_CORVUS_REQ_COMPLETE_KEY),
+									'checkboxesContent' => [
+										[
+											'component' => 'checkbox',
+											'checkboxLabel' => \__('Require complete', 'eightshift-forms'),
+											'checkboxHelp' => \__('Checked indicates an preauthorization. None checked indicates a sale. Note: applicable only for card transactions.', 'eightshift-forms'),
```
Comment (piqusy) [inline src/Integrations/Corvus/SettingsCorvus.php:480]:
```suggestion
											'checkboxHelp' => \__('Checked indicates an pre-authorization. Unchecked indicates a sale. Note: applicable only for card transactions.', 'eightshift-forms'),
```

**Pull request merged**